BioinformaticTools-inator/modules/dna_rna_funcs.py

```diff
@@ -0,0 +1,110 @@
+from typing import Dict, List, Union
+
+dna_dict = {
+    'A': 'T', 'G': 'C', 'T': 'A', 'C': 'G',
+    'a': 't', 'g': 'c', 't': 'a', 'c': 'g'
+    }
+rna_dict = {
+    'A': 'U', 'G': 'C', 'U': 'A', 'C': 'G',
+    'a': 'u', 'g': 'c', 'u': 'a', 'c': 'g'
+    }
+
+
+def is_valid_na(seq: str) -> None:
+    if ('u' in seq or 'U' in seq) and ('t' in seq or 'T' in seq):

```

Comment:
можно сократить условие так:
```suggestion
    if 'u' in seq.lower() and 't' in seq.lower():
```

Reply:
А еще круче перед проверками сделать словарь для seq

```
uniq_seq = set(seq.lower())
```
и тогда проверки будут такими: 
```
if 'u' in uniq_seq and 't' in uniq_seq:
```
У этого есть практическая фича. Программе не придется пробегать по каждому символу строки для проверки, а всего лишь пройтись по словарю, который мы сделаем один раз. И тогда программа будет легче обрабатывать последовательности, если в них будет 100500+ символов